deploy/test/test_in_docker.sh

```diff
@@ -4,7 +4,7 @@ set -xeuo pipefail
 . utils.sh
 
 # Clean up when script completes and fails
-function finish {
```

Comment:
you still have some `function` keywords throughout the code. want to remove those as well?

Reply:
I do? In the `kubernetes-conjur-deploy` folder created on deployment or in the actual secrets-provider?

Reply:
gone now